picatrix/lib/manager.py

```diff
@@ -76,13 +118,50 @@ def deregister_magic(cls, magic_name: str):
       _ = magics_manager.magics.get('cell').pop(magic_name)
 
   @classmethod
-  def get_magic(cls, magic_name: str) -> Callable[[str, str], str]:
+  def get_helper(cls, helper_name: Text) -> Optional[Callable[..., Any]]:
+    """Return a helper function from the registration."""
+    return cls._magics.get(helper_name)
+
+  @classmethod
+  def get_magic(cls, magic_name: Text) -> Callable[[Text, Text], Text]:
     """Return a magic function from the registration."""
     return cls._magics.get(magic_name)
 
   @classmethod
-  def get_magic_info(cls, as_pandas: bool = True) -> Union[
-      pandas.DataFrame, List[Tuple[str, str]]]:
+  def get_helper_info(cls, as_pandas: Optional[bool] = True) -> Union[
+      pandas.DataFrame, List[Tuple[Text, Text]]]:
+    """Get a list of all the registered helpers.
+
+    Args:
+      as_pandas (bool): boolean to determine whether to receive the results
+          as a list of tuple or a pandas DataFrame. Defaults to True.
+
+    Returns:
+        Either a pandas DataFrame or a list of tuples, depending on the
+        as_pandas boolean.
+    """
+    if not as_pandas:
+      return [(x, y.get('help', '')) for x, y in cls._helpers.items()]
```

Comment:
Why not
```
[(name, helper.help) for name, helper in cls._helpers.items()]

```

Reply:
thanks for that, somehow I missed this one